src/qt/upgradeqt.h

```diff
@@ -0,0 +1,62 @@
+#ifndef UPGRADEQT_H
+#define UPGRADEQT_H
+
+#include <string>
+#include <QString>
+
+class UpgradeQt
```

Comment:
i'm able to static most yes. 
3 exceptions to that:
SnapshotMain() since boost thread there with std::bind and uses this
ExtractSnapshot() threaded
DownloadSnapshot() threaded

compile issues with those as I figure i would get something for them